Sources/BraintreeThreeDSecure/BTThreeDSecureV2Provider.m

```diff
@@ -48,14 +48,15 @@ + (instancetype)initializeProviderWithConfiguration:(BTConfiguration *)configura
                                          completion:(BTThreeDSecureV2ProviderInitializeCompletionHandler)completionHandler {
     BTThreeDSecureV2Provider *instance = [self new];
     instance.apiClient = apiClient;
-    instance.cardinalSession = [CardinalSession new];
-    CardinalSessionConfiguration *cardinalConfiguration = [CardinalSessionConfiguration new];
-    if (request.uiCustomization) {
-        cardinalConfiguration.uiCustomization = request.uiCustomization;
-    }
-    CardinalSessionEnvironment cardinalEnvironment = CardinalSessionEnvironmentStaging;
+
+    instance.cardinalSession = (id<BTProxyCardinalSession>)[NSClassFromString(@"CardinalSession") new];
+    id<BTProxyCardinalSessionConfiguration> cardinalConfiguration = (id<BTProxyCardinalSessionConfiguration>)[NSClassFromString(@"CardinalSessionConfiguration") new];
+//    if (request.uiCustomization) {
+//        cardinalConfiguration.uiCustomization = request.uiCustomization;
+//    }
```

Comment:
The UI customization will be fixed in a follow-up PR.

Reply:
Would it make sense to leave a TODO here since it's going into master?

Reply:
Done